ddtrace/settings/integration.py

```diff
@@ -44,8 +43,7 @@ def __init__(self, global_config, name, *args, **kwargs):
         analytics_enabled, analytics_sample_rate = self._get_analytics_settings()
         self.setdefault("analytics_enabled", analytics_enabled)
         self.setdefault("analytics_sample_rate", float(analytics_sample_rate))
-
-        service = get_env(name, "service", default=get_env(name, "service_name", default=None))
+        service = os.getenv(name + "_" + "SERVICE", os.getenv(name + "_" + "SERVICE_NAME", default=None))
```

Comment:
```suggestion
        service = os.getenv("DD_" + name + "_" + "SERVICE", os.getenv("DD_" + name + "_" + "SERVICE_NAME", default=None))
```

? are we missing the `DD_` prefix here?